integrations/lib/multiwoven/integrations/source/amazons3/client.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Multiwoven::Integrations::Source
+  module AmazonS3
```

Comment:
nitpick: We follow rails convention while creating folders in integration gem i.e if the class name is AmazonS3 we name the folder as `amazon_s3`. It won't affect much here but in rails it will throw an error via [zeitwerk](https://github.com/fxn/zeitwerk) gem. Can you please update the folder name

Reply:
This is convention in the integration gem also. Let's follow that pls